src/ansys/fluent/core/session.py

```diff
@@ -152,6 +152,7 @@ def __init__(
             event_type,
             get_zones_info,
         )
+        self.register_finalizer_cb = fluent_connection.register_finalizer_cb
```

Comment:
@prmukherj If exposing this, we have to give it a better named. The problem is that "cb" is a meaningless abbreviation. What naming format are we using for similar methods elsewhere?

Reply:
"register_finalizer_callback" should be fine I think, as other methods are named "register_callback"

Reply:
@prmukherj Yes, that's meaningful.